dashboard/app/models/script_level.rb

```diff
@@ -66,6 +66,48 @@ def stage_total
     stage.script_levels.to_a.count
   end
 
+  def summarize
+    if level.unplugged?
+      kind = 'unplugged'
+    elsif assessment
+      kind = 'assessment'
+    else
+      kind = 'puzzle'
+    end
+
+    summary = {
+        id: level.id,
```

Comment:
I agree - & I don't think it's used much any more tbh.  As far as I remember it's only in there so that we can highlight the current level (but don't hold me to that)  That was harder to do before you adjusted all the scripts to have valid stage/level/etc, I can look into it if you think it should hold up the checkin